curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-algorithmic-thinking-by-building-a-number-sorter/6407c4abf5be6d07d8c12ade.md

```diff
@@ -0,0 +1,327 @@
+---
+id: 6407c4abf5be6d07d8c12ade
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+You'll need to take an action for each of the elements in the array. Remember that arrays have a specific method for this.
+
+Use the `forEach()` method, passing an empty callback which takes `num` and `i` as the parameters.
```

Comment:
Use the `forEach()` method, and pass in an empty callback that takes `num` and `i` as the parameter. 

It's more readable that way.